packages/api/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@comet-brevo-module/api",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet-brevo-module/",
+        "directory": "packages/api/"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "bin": {
+        "comet-api-generator": "bin/comet-api-generator.js"
+    },
+    "files": [
+        "bin/**/*.js",
```

Comment:
```suggestion
```

Reply:
I removed this in c94ff3cacc315c168cf437a2f8d6b45a0f7c96b6